support/Pipelines/Bbh/Inspiral.yaml

```diff
@@ -139,6 +139,7 @@ DomainCreator:
 Evolution:
   InitialTime: *InitialTime
   InitialTimeStep: 0.0002
+  MinimumTimeStep: 1e-6
```

Comment:
Was there a reason to change this from the hardcoded value 1e-9?

Reply:
I don't know what a good value is.  SpEC uses 1e-5, but because of different time stepper choices we expect to be a bit smaller than SpEC in some cases.  The 1e-9 was chosen as "about 2^-31", which is still there except changed to exactly 2^-31, and is a relative limit rather than an absolute one.

What minimum time step do we typically see in BBH runs?

Reply:
Here's the min/max time step for a high resolution BBH from merger to ringdown. It reaches down to 1e-5. This run was with relative tolerance 1e-6 and absolute tolerance 1e-8, but @geoffrey4444 is trying runs with 100x tighter tolerances. I'd be worried that 1e-6 was too large near merger

![image](https://github.com/user-attachments/assets/ff3f59ce-b76e-494c-b2ca-c2ae17f20745)

Reply:
I'm not sure if spectre is doing this, but in spec the time step near merger gets small because of the control system, not the error stepping. I guess we don't have that coupling here?  I don't know what to expect with 100x tighter tolerance, but assuming the solution is reasonably well-behaved then I would expect only a ~2-3 smaller time step for a 4th-order method. If we are doing higher order AMPC then it could be even less than that.

Reply:
No we don't have a coupling between the time step and control system. @wthrowe @nilsdeppe What do you think about 1e-7 then as the value we have for the BBH/SingleBH LTS executables? 1e-6 seems ok for other systems (especially ones using GTS)

Reply:
After seeing the above plot, my inclination is to drop the minimum to 1e-7 (just haven't gotten to it yet).  I also don't expect a very large change from the higher tolerance, but if we're measuring time steps of ~1e-5 on an easy case we'll probably get a bit lower on more extreme systems.  There may also be smaller steps that we don't see in the plot, since we don't output every time step.

Reply:
That sounds good to me!

Reply:
Changed.